application/electron/src/tools/fs.ts

```diff
@@ -276,3 +278,16 @@ export function copyFolder(source: string, dest: string) {
         });
     }
 }
+
+export function isHidden(file: string): Promise<boolean> {
+    return new Promise((resolve) => {
+        fswin.getAttributes(file, (result: any) => {
+            const key = 'IS_HIDDEN';
+            if (result && result[key] !== undefined) {
+                return resolve(result[key]);
+            } else {
+                return resolve(false);
+            }
+        });
```

Comment:
Can we make it crossplatform? I mean cover here and windows and Linux based OS? I mean, just move functionality from service.file.opener 
```
private hidden(file: string): Promise<boolean> {
        return new Promise((resolve, reject) => {	        return new Promise((resolve) => {
            fs.access(file.path, fs.constants.R_OK, (err) => {	            const ops = os.getPlatform();
                if (err) {	            if (ops === os.EPlatforms.win32 || ops === os.EPlatforms.win64) {
                    return reject(new Error(this._logger.warn(`Fail to read file(s) due to missing access rights for ${file.path}`)));	                isHidden(file).then((hid: boolean) => {
                }	                    return resolve(hid);
                return resolve();	                }).catch((err: Error) => {
            });	                    this._logger.warn(`Fail to check if ${file} is hidden due to error: ${err.message}`);
                    return resolve(false);
                });
            } else {
                return resolve((/(^|\/)\.[^\/\.]/g).test(file));
            }
        });	        });
    }	    }
``` 
to here. It would be more logical. 
And in service.file.opener you will just call this method without any OS checks